oaknational/Oak-Web-Application
# Feat/pupil 881/filter skip button

## Description

Music year: 1982

- List of changes
- add to oakpupiljourneyunitsfilter Skip to results button
- Display skip button when keyboard tabbing through the page to allow users to skip direct to the results.
- We don't have as many filter options as in the curric example so make this conditional on a minimum number of filter items (3).

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2956--oak-web-application.netlify.thenational.academy/pupils/programmes/english-primary-year-6/units
2. tab to the filter then click the "skit to results" button
3. You should see the tab button when and only when tabbed to

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
